packages/opentelemetry-tracing/src/export/InMemorySpanExporter.ts

```diff
@@ -25,15 +25,16 @@ import { ExportResult } from '@opentelemetry/core';
  */
 export class InMemorySpanExporter implements SpanExporter {
   private _finishedSpans: ReadableSpan[] = [];
-  private _stopped = false;
+  protected _stopped = false;
```

Comment:
please add some doc

Reply:
@obecny what sort of doc are you looking for? 

do you want a comment on why this is protected? that seems like something that would quickly get out of date with future usages, outside of the obvious fact it is now available to classes extending the InMemorySpanExporter.

or are you looking for something else?

Reply:
If something is protected its meaning should be documented so that when someone subclasses this they know what the property represents. It could be argued it is obvious in this case, but it is good practice in general.

Reply:
OK. i don't see that done for any other protected members (non-function) in the code base and somewhat debate the value here but am happy to go along with the standards y'all are looking for.

Reply:
@michaelgoin I agree we haven't been great at being consistent about this, and in many cases the value is questionable anyways.